contracts/test-integration/BlsProvider.test.ts

```diff
@@ -201,36 +201,58 @@ describe("BlsProvider", () => {
       value: parseEther("1"),
     });
 
+    const expectedToAddress = "0x689A095B4507Bfa302eef8551F90fB322B3451c6"; // Verification Gateway address
+    const expectedFromAddress = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"; // Aggregator address (Hardhat account 0)
+
     // Act
     const transactionReceipt = await blsProvider.waitForTransaction(
       transactionResponse.hash,
       1,
-      10,
+      20,
     );
 
     // Assert
-    // TODO: bls-wallet #412 Update values returned in bundle receipt to more closely match ethers transaction response
+    const expectedBlockNumber = await blsProvider.getBlockNumber();
     expect(transactionReceipt).to.be.an("object").that.deep.includes({
```

Comment:
These assertions are more verbose but feel like the tradeoff is worth it unless anyone has suggestions on making them more concise.

If someone edits the return value in the BundleService, either by removing a property or changing property types, the test will fail. Note the test does not fail if someone adds an invalid property to the bundle receipt see comment